apps/simple-trading-app/src/app/components/simple-market-list/simple-market-toolbar.spec.tsx

```diff
@@ -3,46 +3,57 @@ import { useLocation, useRoutes, BrowserRouter } from 'react-router-dom';
 import { render, screen, fireEvent, waitFor } from '@testing-library/react';
 import { MockedProvider } from '@apollo/react-testing';
 import SimpleMarketToolbar from './simple-market-toolbar';
-import type { MockedResponse } from '@apollo/client/testing';
-import type { MarketFilters } from './__generated__/MarketFilters';
-import { FILTERS_QUERY } from './data-provider';
-import filterData from './mocks/market-filters.json';
-import { act } from 'react-dom/test-utils';
+import type { SimpleMarkets_markets } from './__generated__/SimpleMarkets';
+import { markets as filterData } from './mocks/market-filters.json';
 
 describe('SimpleMarketToolbar', () => {
-  const filterMock: MockedResponse<MarketFilters> = {
-    request: {
-      query: FILTERS_QUERY,
-    },
-    result: {
-      data: filterData as unknown as MarketFilters,
-    },
-  };
-
   const WrappedCompForTest = () => {
     const routes = useRoutes([
       {
         path: '/',
-        element: <SimpleMarketToolbar />,
+        element: (
+          <SimpleMarketToolbar
+            data={filterData as unknown as SimpleMarkets_markets[]}
```

Comment:
Question (non-blocking): why is it necessary to cast as unknown as well?

Reply:
might be because it's imported from a json file, which doesn't have associated types

Reply:
yes, cast to unknown is redundant if just after we cast to proper type. Thanks for the catch. Let's stay with version with plain json mock file, should have better readability.